cross/x86_64-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:18.04
```

Comment:
Not exactly sure what this and `Cross.toml` are. Seems `nearcore` related and they were also removed from this repo, maybe we can delete them now?

Reply:
it's not `nearcore` related. This is used to build the binary. Previously adopted from https://github.com/near/near-lake-indexer (which is adopted from `near-cli-rs`, it's complicated :) )

Reply:
Yeah, since we're going to build the binaries in GCB we should drop this file

Reply:
Cross can be removed for near-lake-indexer as we basically support only x86_64-linux target (make sure CI building the binary release also does not use it). Cross is used for cross-target compilation of near-cli-rs binaries as we support ARM/x86/x86_64 Linux/macOS/Win there.

Reply:
Thanks for the context 👍🏽 I've removed all `Cross` references